content/en/integrations/guide/source-code-integration.md

```diff
@@ -84,9 +78,20 @@ export DD_TAGS="git.commit.sha:<GIT_COMMIT_SHA> git.repository_url=<REPOSITORY_U
 {{% /tab %}}
 {{< /tabs >}}
 
-### Upload your git metadata
+### Configure repositories
+
+{{< tabs >}}
+{{% tab "Configure a GitHub App" %}}
```

Comment:
Same here, it might be possible to re-use the same tabs and have them synchronized ?

Reply:
Yes it works!